projects/dsp-ui/src/lib/action/pipes/array-transformation/key.pipe.ts

```diff
@@ -0,0 +1,26 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+/**
+ * This pipe can be used for "for loops", in the case of an array with non-numeric indexes.
+ * It returns the key and the value(s). In the example below the {{item.key}} contains the index value
+ * and the {{item.value}} contains the value(s).
+ *
+ * When the value is an object with name and label, you get them with:
+ * {{item.value.name}} and {{item.value.label}}
+ *
+ */
+@Pipe({
+    name: 'dspKey'
+})
+export class KeyPipe implements PipeTransform {
+
+    transform(value: any, args?: any): any {
```

Comment:
@kilchenmann will check if this pipe is still needed or if the built-in pipe is enough

Reply:
André investigated and found that this pipe was created before there was a built-in pipe for this. This pipe has been removed in 3c8b2ae in favor of the built-in angular KeyValue pipe. The only places that use this pipe are in the viewer module list- and grid-views but this will be refactored to use the built-in pipe so I don't see this as a breaking change.

Reply:
less code, less maintenance ;-)